client/mixins/flash.js

```diff
@@ -0,0 +1,23 @@
+import { FLASH_MESSAGE, CLEAR_FLASH_MESSAGE } from '@store/flash/mutations';
+import Uuid from 'uuid/v4';
+export default {
+  computed: {
+    flashMessages() {
+      return this.$store.state.flash.messages;
+    }
+  },
+  methods: {
+    flash(message, type = 'success') {
+      const id = Uuid();
```

Comment:
Codacy found an issue: [A function with a name starting with an uppercase letter should only be used as a constructor.](https://app.codacy.com/manual/sam-spain/full-stack-js/pullRequest?prid=6207934)